helper_scripts/benchmark.py

```diff
@@ -111,7 +136,10 @@ def main():
 
         print('Starting consumer')
         consumer_settings = requests.get('http://consumer:3000/setting').json()
-        response = requests.post('http://consumer:3000/setting', json=consumer_settings)
+        new_consumer_settings = set_consumer_ratios(consumer_settings,
+                                                    prefer_cheap = random.randint(4, 7),
+                                                    cheapest_best_quality = random.randint(2, 4))
```

Comment:
The benchmark tool should make simulations as reproducible as possible.
That's why I prefer non-random number here.

Reply:
True. This setting was thought to be a better setting for the seminar in order to avoid overfitting, but it's not ideal for a reproducible benchmark script.

I'll change it.